src/packages/@ncigdc/containers/ConsequencesTable.js

```diff
@@ -131,6 +136,7 @@ const ConsequencesTableComponent = compose(
       style={{ width: '100%', minWidth: 450 }}
       data={dataRows}
       prefix={paginationPrefix}
+      entityName={'entries'}
```

Comment:
another way to look at it is that it's a table of entries that happen to have unique transcripts, non-unique consequence types and aa variants